klippy/extras/servo.py

```diff
@@ -28,28 +28,51 @@ def __init__(self, config):
         self.gcode.register_mux_command("SET_SERVO", "SERVO", servo_name,
                                         self.cmd_SET_SERVO,
                                         desc=self.cmd_SET_SERVO_help)
-    def set_pwm(self, print_time, value):
+
+        self.initial_pwm_value = -1
+        #Check to see if an initial angle or pulse width is configured and
+        # set it as required
+        initial_angle = config.getfloat('initial_angle', -1)
```

Comment:
This should have minval and maxval settings to ensure the config is sane.

Reply:
_get_pwm_from_angle does a min and max check (0 and max_angle) before sorting out the pwm value. (as does _get_pwm_from_pulse_width) so in either case the values entered in the config file are sanitized before being converted to pwm values.